client/app.js

```diff
@@ -0,0 +1,55 @@
+const form = document.querySelector('form')
+
+
+
+form.addEventListener('submit', getResults)
+document.querySelector('#lucky').addEventListener('click', getOneResult)
+
+async function getResults(e) {
+    e.preventDefault()
+    let search = document.querySelector('#search-bar').value
+    let res = await fetch(`http://localhost:3000/${search}`)
+    let resultsList = await res.json()
+    console.log(resultsList)
+    if( resultsList.error){
+        document.body.style.display = 'block'
+        document.querySelector('.homepage').style.display = 'none'
```

Comment:
by setting this to none the logo and search bar disappear once u make a search request